openapi-filter.js

```diff
@@ -30,20 +31,22 @@ let argv = require('yargs')
     .number('lineWidth')
     .alias('l', 'lineWidth')
     .describe('lineWidth','max line width of yaml output')
-    .default('lineWidth',-1)
+    .default('lineWidth',80)
```

Comment:
Making the max line width 80 here could be a breaking change for people. Can we not?